### PR TITLE
Auto-update minor and patch for some dependencies

### PR DESCRIPTION
**Pull request by fhinkel**
Changing a few hand-picke dependencies where we trust the minor and
patch updates. If dependency versions are fixed, we are drowning in
RenovateBot PRs.

### CODE REVIEW

File: appengine/loopback/package.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
     "loopback-component-explorer": "5.4.0",
     "serve-favicon": "2.5.0",
     "strong-error-handler": "3.2.0",
-    "loopback-datasource-juggler": "4.5.3",
+    "^loopback-datasource-juggler": "4.5.3",
```
Comment (alexander-fenster) [inline appengine/loopback/package.json]: is this some special syntax, or a typo? I expected `"loopback-datasource-juggler": "^4.5.3"`

Comment (fhinkel) [reply appengine/loopback/package.json]: Yikes, thanks for noticing.

Comment (fhinkel) [reply appengine/loopback/package.json]: How did the tests pass though? 🙈

**Comment by fhinkel** [comment]
@alexander-fenster fixed the issue, please have another look.

File: appengine/analytics/package.json [hunk 2]
```diff
@@ -19,7 +19,7 @@
   },
   "dependencies": {
     "express": "4.16.4",
```
Comment (JustinBeckwith) [inline appengine/analytics/package.json]: Would it be bad to try brining express up to `^` while you're in here?

Comment (fhinkel) [reply appengine/analytics/package.json]: Only changing the deps that I recently updated, otherwise I'll have too many breaking changes.

**Pull request merged**